frontend/src/components/AssessmentCard.js

```diff
@@ -0,0 +1,69 @@
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import { LinearProgress } from '@mui/material';
+
+import QuestionWrapper from './QuestionWrapper';
+import { boxStyles } from './boxStyles';
+import { useEffect, useState } from 'react';
+
+const AssessmentCard = ({ completeAssessment, responses, setResponses }) => {
+  const [activeQuestionIndex, setActiveQuestionIndex] = useState(0);
+  const [assessmentData, setAssessmentData] = useState([]);
+
+  function fetchAssessmentData() {
+    fetch('http://localhost:3002/api/assessments/data')
+      .then(response => response.json())
+      .then(data => setAssessmentData(data))
+      .catch(err => console.error(err));
+  }
+
+  useEffect(() => {
+    fetchAssessmentData();
+  }, []);
+
+  if (!assessmentData?.content?.sections) return null;
+
+  // Not sure if there would be more sections
+  const section = assessmentData.content.sections[0];
```

Comment:
I worked under the assumption that there's only one section as adding even another one would disrupt the current flow and I imagine is out of the scope of the project. However, if that were to be the case, I imagine I'd create another functional component called `SectionWrapper` which would be iteratively called in here.